docs/new_horizen/04-attestation_mechanism/03-initial_functionality.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Attestation Mechanism Initial Functionality
+---
+
+# Attestation Mechanism Initial Functionality
+
+## Initial Functionality
```

Comment:
Do we want to call it "Initial Functionality"? It feels like a term we use internally to dictate that this is our first phase.

Reply:
Fixed offline